docs/content/Modpacks/Materials-and-Elements/Tool_Properties.md

```diff
@@ -58,41 +70,65 @@ The builder has the same arguments as the constructor, and can have chained meth
-    event.create('aluminfrost')
-        .ingot()
-        .color(0xadd8e6).secondaryColor(0xc0c0c0).iconSet(GTMaterialIconSet.DULL)
-        .toolStats(
-            $ToolProperty.Builder.of(1.8, 1.7, 700, 3,
-                [
-                    GTToolType.SWORD,
-                    GTToolType.PICKAXE,
-                    GTToolType.SHOVEL,
-                ]
-            )
-            .unbreakable()
-            .addEnchantmentForTools(silk_touch, 1)
-            .build()
-        ) 
-});
-```
+=== "JavaScript"
+    ```js title="example_tool_material.js"
+    GTCEuStartupEvents.registry('gtceu:material', event => {
+        event.create('aluminfrost')
+            .ingot()
+            .color(0xadd8e6).secondaryColor(0xc0c0c0).iconSet(GTMaterialIconSet.DULL)
+            .toolStats(
+                $ToolProperty.Builder.of(1.8, 1.7, 700, 3,
+                    [
+                        GTToolType.SWORD,
+                        GTToolType.PICKAXE,
+                        GTToolType.SHOVEL,
+                    ]
+                )
+                .unbreakable()
+                .addEnchantmentForTools(silk_touch, 1)
+                .build()
+            ) 
+    });
+    ```
+=== "Java"
+    ```java title="example_tool_material.java"
+            public static Material ALUMINFROST;
+            ALUMINFROST = new Material.Builder(
+                your_mod_id.id("aluminfrost"))
+                .color(0xadd8e6).secondaryColor(0xc0c0c0).iconSet(MaterialIconSet.DULL)
+                .toolStats(ToolProperty.Builder.of(7.0F, 6.0F, 2560, 3)
+                        .types(GTToolType.AXE, GTToolType.DRILL_EV)
+                        .unbreakable().enchantability(21).enchantment(SILK_TOUCH, 1).build())
+                .buildAndRegister();
+    ```
 
 You can also change the tool property of a GT material that already has a tool property. You do, however, have to remove the current tool property as it is immutable.
-```js title="tool_replacement.js"
-GTCEuStartupEvents.materialModification(event => {
-    if (GTMaterials.Iron.hasProperty($PropertyKey.TOOL)) {
-        GTMaterials.Iron.removeProperty($PropertyKey.TOOL);
+=== "JavaScript"
+    ```js title="tool_replacement.js"
+    GTCEuStartupEvents.materialModification(event => {
+        if (GTMaterials.Iron.hasProperty($PropertyKey.TOOL)) {
+            GTMaterials.Iron.removeProperty($PropertyKey.TOOL);
+        }
+        GTMaterials.Iron.setProperty($PropertyKey.TOOL, 
+            $ToolProperty.Builder.of(180, 5.9, 2147483647, 6,
+            [
+                GTToolType.SOFT_MALLET,
+                GTToolType.DRILL_LV
+            ]
+        ).build());
+    });
+    ```
+=== "Java"
+    ```java title="tool_replacement.java"
+    public static void modifyMaterials() {
+        if (GTMaterials.Iron.hasProperty($PropertyKey.TOOL)) {
+            GTMaterials.Iron.removeProperty($PropertyKey.TOOL);
+        }
+        GTMaterials.Iron.addFlags(PhoenixMaterialFlags.GENERATE_NANITES);
```

Comment:
This isn't replacing the tool property. Also, we should not delete iron pickaxes in our example, see #3913